### PR TITLE
website/docs: add . in https://netbird.company*

### PR DESCRIPTION
**Pull request by Marcus1Pierce**
## Details

From the documentation https://docs.netbird.io/selfhosted/identity-providers#authentik, the domain must have a . in https://netbird.company.* or you will experience a redirect error.

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by tanberry** [comment]
Thanks for this PR @Marcus1Pierce ! So both the period (.) and the asterisk (*) needed to be added, for the Redirect URI, right?
Screenshot 2024-11-26 at 3 24 57 PM

**Comment by tanberry** [comment]
> Thanks for this PR @Marcus1Pierce ! So both the period (.) and the asterisk (*) needed to be added, for the Redirect URI, right? Screenshot 2024-11-26 at 3 24 57 PM

oh, wiat, I see... it's the second line you corrected, the asterisk was already there, you added the period. Got it. Thanks so much!

File: website/integrations/services/netbird/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The following placeholders will be used:
 5. In the provider settings, add the following redirect URLs under **Redirect URIs/Origins (RegEx)**:
     ```
     https://netbird.company
-    https://netbird.company*
+    https://netbird.company.*
     http://localhost:53000
     ```
```
[the remark this answers is not in the document]
Comment (Marcus1Pierce) [reply website/integrations/services/netbird/index.md:36]: I tried with https://netbird.company/.* or https://netbird.company.* and it seems there are no issues. But if I don't add .* to the regex (ex. https://netbird.company/* or https://netbird.company*), for some reason, the Redirect URI Error always appears as shown in the image.

image

**Review by tanberry — APPROVED** [review]
Thanks!~

Oh. Oops, I just saw @BeryJu 's comment. Sorry, we will get back to this one...

File: website/integrations/services/netbird/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tanberry) [reply website/integrations/services/netbird/index.md:36]: OK, thanks @Marcus1Pierce for your patience. In the last release, we updated the Redirect URI fields such that you can select Strict or Regex format. So exactly as @BeryJu 's suggested change shows, we want users to indicate that it is Regex if they use the `https://netbird.company/.*` format.

Here's what that field looks like now:
Screenshot 2024-11-26 at 5 13 35 PM

[the remark this answers is not in the document]
Comment (Marcus1Pierce) [reply website/integrations/services/netbird/index.md:36]: @tanberry I tried as suggested by @BeryJu and everything works fine.

image

Or as suggested on the website https://docs.netbird.io/selfhosted/identity-providers#authentik (all using regex) as below also works fine.

image

I don't know which is the best, but from what I tried, everything works fine.

[the remark this answers is not in the document]
Comment (BeryJu) [reply website/integrations/services/netbird/index.md:36]: Using all regex is insecure without escaping (which is why we added the configuration options for the comparison mode)

[the remark this answers is not in the document]
Comment (tanberry) [reply website/integrations/services/netbird/index.md:36]: Hi again @Marcus1Pierce good to hear this works. Please make the change to the file with @BeryJu 's suggestion, run the linters again (`make website`), and push again, and then we can merge.

[the remark this answers is not in the document]
Comment (Marcus1Pierce) [reply website/integrations/services/netbird/index.md]: @tanberry I have edited it again according to the suggestion by @BeryJu.

[the remark this answers is not in the document]
Comment (tanberry) [reply website/integrations/services/netbird/index.md]: Thanks @Marcus1Pierce , we're getting there. Can you please run `make website` again, and re-push? Our dependabot bumped the version of the linter we use, and this PR is failing on that check. NOTE: you might need to first run `make website-install` in order to update the build tools locally, then run `make website`. Then push to the PR again.

Let us know if any problems.

[the remark this answers is not in the document]
Comment (Marcus1Pierce) [reply website/integrations/services/netbird/index.md]: @tanberry Sorry for the late update. I have tried running `make website-install` and then `make website`, but the following error occurred:

```
[ERROR] Error: Unable to build website for locale en.
    at tryToBuildLocale (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:78:15)
    at async /mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:34:9
    ... 4 lines matching cause stack trace ...
    at async file:///mnt/d/Visual%20Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/bin/docusaurus.mjs:44:3 {
  [cause]: Error: Docusaurus found broken links!

  Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
  Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

  Exhaustive list of all broken links found:
  - Broken link on source page path = /docs/add-secure-apps/flows-stages/flow/examples/flows:
     -> linking to /blueprints/example/flows-enrollment-2-stage.yaml
     -> linking to /blueprints/example/flows-enrollment-email-verification.yaml
     -> linking to /blueprints/example/flows-login-2fa.yaml
     -> linking to /blueprints/example/flows-login-conditional-captcha.yaml
     -> linking to /blueprints/example/flows-recovery-email-verification.yaml
     -> linking to /blueprints/example/flows-unenrollment.yaml
  - Broken link on source page path = /docs/users-sources/user/invitations:
     -> linking to /blueprints/example/flows-enrollment-2-stage.yaml

      at throwError (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/logger/lib/logger.js:80:11)
      at reportBrokenLinks (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/server/brokenLinks.js:250:47)
      at handleBrokenLinks (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/server/brokenLinks.js:282:5)
      at executeBrokenLinksCheck (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js:91:47)
      at /mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js:70:67
      at Object.async (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/logger/lib/perfLogger.js:42:47)
      at buildLocale (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js:70:31)
      at async runBuildLocaleTask (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:93:5)
      at async /mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:74:13
      at async tryToBuildLocale (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:70:9)
      at async /mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:34:9
      at async mapAsyncSequential (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/utils/lib/jsUtils.js:21:24)
      at async Command.build (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/build/build.js:33:5)
      at async Promise.all (index 0)
      at async runCLI (/mnt/d/Visual Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/lib/commands/cli.js:56:5)
      at async file:///mnt/d/Visual%20Studio/GitHub/Marcus1Pierce/authentik/website/node_modules/@docusaurus/core/bin/docusaurus.mjs:44:3
}
[INFO] Docusaurus version: 3.6.2
Node version: v18.19.1
make: *** [Makefile:250: website-build] Error 1
```

**Review by tanberry — APPROVED** [review]
Thanks again @Marcus1Pierce sorry for the build drama. ;-)

File: .gitignore [hunk 2]
```diff
@@ -209,3 +209,5 @@ source_docs/
 
 ### Golang ###
 /vendor/
+/.vs
+/node_modules
```
Comment (rissson) [inline .gitignore:213]:
```suggestion
```

**Comment by Marcus1Pierce** [comment]
@rissson @tanberry Sorry, it seems I made a mistake. In the regex, it should be `https://netbird.company/.*` instead of `https://netbird.company/.\*`. Could you please help to change it? Once again, I apologize for not checking it thoroughly. In the future, I will open issues instead of pull requests to prevent this from happening again. I'm really sorry. I am currently unable to open the editor to fix it.

**Comment by rissson** [comment]
No worries @Marcus1Pierce, mistakes happen. I just opened https://github.com/goauthentik/authentik/pull/12284 to fix this. Feel free to keep making PRs, they are always welcomed!

**Comment by tanberry** [comment]
Yes, thank you again @Marcus1Pierce, we really appreciate the PRs, and I apologize for my messy handling of this one! As @rissson says, PRs are always welcome! Thanks again.

**Pull request merged**